src/OpenFOAM/CMakeLists.txt

```diff
@@ -609,4 +609,14 @@ target_link_libraries(OpenFOAM PUBLIC Pstream)
 target_include_directories(OpenFOAM PUBLIC
   .
 )
+
+find_package(ZLIB REQUIRED)
```

Comment:
We have this here: https://github.com/openfoamdotcom/openfoam/blob/d29f00600d42321444580c3d49d167045e01bce4/CMakeLists.txt#L25C1-L25C28